operations/validate_csm_health.md

```diff
@@ -55,6 +55,16 @@ If running these checks after the initial CSM install, then to find details on c
 
 All platform health checks are expected to pass. Each check has been implemented as a [Goss](https://github.com/aelsabbahy/goss) test which reports a `PASS` or `FAIL`.
 
+Before running health checks, restart `goss-servers` on all NCNs to ensure the correct tests are run on each node. This is necessary because of a timing issue on the nodes that has been fixed in CSM 1.6.1.
+
+(`ncn-m#` or `pit#`) Restart `goss-servers`.
+
+```bash
+ncn_nodes=$(grep -oP "(ncn-s\w+|ncn-m\w+|ncn-w\w+)" /etc/hosts | sort -u | tr -t '\n' ',')
```

Comment:
Per our Slack discussion, the command to get the NCN node list should be split between the PIT scenario and the non-PIT scenario.

Reply:
I have added the PIT scenario.